pocs/utils/__init__.py

```diff
@@ -49,7 +49,7 @@ def flatten_time(t):
 
 
 # This is a streamlined variant of PySerial's serialutil.Timeout.
-class Timeout(object):
+class Timer(object):
```

Comment:
I suppose this is a good change since I see we already have an error type called Timeout. However, note also that we're using threading.Timer in some places. If you are concerned by that, you could consider calling this TimeLimit, which is not a symbol that we use.

Reply:
`CountdownTimer`?